__tests__/endpoint.test.js

```diff
@@ -332,3 +332,72 @@ describe('POST/api/articles/:article_id/comment', () => {
   });
 
 })
+
+describe('PATCH/api/articles/:article_id', () => {
+  test('PATCH:200 sends a single article object to the client with vote increment', () => {
+
+    return db.query('SELECT * FROM articles WHERE article_id = 1').then ((result)=>{
```

Comment:
Instead of querying the database here, we could just give the actual number of votes.